src/hooks/useDrivesContainer.ts

```diff
@@ -37,14 +36,14 @@ export function driveToBaseItems(
         id: driveID,
         label: drive.name,
         path: driveID,
-        type: ItemType.LocalDrive,
```

Comment:
We no longer export these types?

Reply:
nope, this is part of this refactor: https://github.com/powerhouse-inc/design-system/pull/111